concepts/comparisons/introduction.md

```diff
@@ -1,2 +1,14 @@
-#TODO: Add introduction for this concept.
+# Introduction
 
+A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the values of two operands and returns `True` or `False` based on whether the `comparison` condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, and `"!="`. They all have the same priority (which is higher than that of the Boolean operations)

```

Comment:
```suggestion
A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the values of two operands and returns `True` or `False` based on whether the `comparison` condition is met.
The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, and `"!="`.
They all have the same priority (which is higher than that of the Boolean operations)
```

As per the [build markdown][1] style guide; single sentence per line.

[1]: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md